src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterStrategyConfiguration.java

```diff
@@ -13,6 +12,7 @@
     private Boolean     utilizeReservedInstances;
     private Integer     drainingTimeout;
     private Integer     spotPercentage;
+    private Boolean     utilizeCommitments;
```

Comment:
fallbackToOd parameter is missing.
utilizeCommitments parameter is not present in the strategy object.

Reply:
fallbackToOd parameter is already there, here is the reference: 
https://github.com/spotinst/spotinst-sdk-java/blob/qabansi_aut_6598_sdk_ocean_ecs/src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterStrategyConfiguration.java
utilizeCommitments parameter was present in the config JSON we get on UI. Removed now, to align with Postman Request parameters.